modular_stonehedge/licensed-death-rattler/Death-Rattler/code/Death-Rattler.dm

```diff
@@ -0,0 +1,158 @@
+// This code is part of Death-Rattler
+// Copyright (C) 2024 Moribund/Luctus-Comptus
+// Unauthorized use, modification, or distribution of this code is prohibited without copyright holders approval. See LICENSE.md
+
+// I hate doing this. basically if you are anyone but ratwood/gyran DM me first and you can use whatever the fuck your heart wants from this. not only did he brag about theft and how easy it is he tryed to pull this same shit.
+
+
+
+
+/obj/item/storage/fancy/shhig
+	name = "Shhig Brand Zigs"
+	desc = "Dr. V's Shhig's; much like the image of serpents these zigs are synonymous with both healing and killing. Your life expectancy isnt very high anyway."
```

Comment:
```suggestion
	desc = "Dr. V's Shhig's: much like the image of serpents, these zigs are synonymous with both healing and killing. Your life expectancy isn't very high anyway."
```

Reply:
also I assume this is about the actual snakes compared to the rod of Asclepius, but that's not 100% clear to the average reader, i'd think, so it just comes off a _little_ strange